src/ManualClient.py

```diff
@@ -98,7 +98,8 @@ async def connection_closed(self):
     def suggested_game(self) -> str:
         if self.game:
             return self.game
-        return Utils.persistent_load().get("client", {}).get("last_manual_game", "Manual_{\"game\" from game.json}_{\"creator\" from game.json}")
+        from .Game import game_name  # This will at least give us the name of a manual they've installed
+        return Utils.persistent_load().get("client", {}).get("last_manual_game", game_name)
```

Comment:
Yeah, this will work out better than the hint text. Since more people lately have only used Builder and have been doing `Manual_{Manual_Blah_Me}_Me` due to not knowing what `game.json` is.